map_frame: map
local_frame: local
body_frame: body
imu_topic: /livox/imu
livox_topic: /livox/lidar

local_rate: 20.0
loop_rate: 1.0

lio_builder:
  scan_resolution: 0.5
  map_resolution: 0.5
  point2plane_gain: 1000.0
  plane2plane_gain: 100.0
  map_capacity: 5000000
  grid_capacity: 20
  extrinsic_est_en: false
  align_gravity: true
  near_mode: 1
  imu_ext_rot: [1, 0, 0, 0, 1, 0, 0, 0, 1]
  imu_ext_pos: [0.05512, 0.02226, -0.0297]

loop_closure:
  activate: true
  rad_thresh: 0.2
  dist_thresh: 1.0
  time_thresh: 30.0
  loop_pose_search_radius: 15.0
  loop_pose_index_thresh: 25
  submap_resolution: 0.5
  submap_search_num: 25
  loop_icp_thresh: 0.3
  z_prior: false